hackathon_site/dashboard/frontend/src/api/types.ts

```diff
@@ -0,0 +1,44 @@
+/** Generics */
+export interface APIListResponse<T> {
+    count: number;
+    next: string | null;
```

Comment:
is this like an optional string?

Reply:
This just says that `next` is either a string or null (which is what the API will return). You can make a [union type](https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html#union-types) by chaining types with `|`.

The normal way to mark something as optional is to do `next?: string`, which pretty much means `next: string | undefined`.